src/Apps/Legion/Components/WizardShell/Internal/SaveStep/SaveStep.styles.ts

```diff
@@ -9,7 +9,12 @@ const classNames = {
 // export const SAVESTEP_CLASS_NAMES = classNames;
 export const getStyles = (_props: ISaveStepStyleProps): ISaveStepStyles => {
     return {
-        root: [classNames.root],
+        root: [
+            classNames.root,
+            {
+                height: '440px'
```

Comment:
is this the same height assumption as the other screen? Could we move that to a constant so we can see all the references if/when we replace it?